### PR TITLE
Feature/dx 971 pba tables

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -111,7 +111,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
-    rack (2.1.4)
+    rack (1.6.13)
```
Comment (payex-dev) [inline Gemfile.lock:114]: Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5842155)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5842155&bid=19128961)

File: _includes/pba-tables.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+| `Reference`                     | `Reference`             | {% if documentation_section=="card" or documentation_section=="payment-menu" or documentation_section=="checkout" %}transaction.number if DirectDebit {% endif %} |
+| `Swedbank Account Number`       | `SwedbankAccountNo`     |                                                                                                         |
+| `Referenced Transaction Number` | `ReferencedTransaction` | {% unless operation_title=="reversal" %}transaction.number if reversed later{% endunless %}             |
+| `Sales Channel`                 | `SalesChannel`          |                                                                                                         |
+| `Brand`                         |                         |                                                                                                         |
+| `Point Of Sale`                 |                         |                                                                                                         |
+
+{%- endcapture -%}
+{{- code -}}
```
[the remark this answers is not in the document]
Comment (marinafrydenb) [reply _includes/pba-tables.md]: fixed!

File: _includes/settlement-reconciliation.md [hunk 3]
```diff
@@ -211,6 +213,17 @@ the reconciliation file.
 *   `capture.transaction.number` returned from Swedbank Pay is equal to
     `TransactionNo` in reconciliation file.
 
+Below you will see the API mapping tables to the fields in the PBA reports for
+`Create payment`, `Capture` and `Reversal`.
```
Comment (asbjornu) [inline _includes/settlement-reconciliation.md:217]: This description should be different based on the `documentation_section` as `swish` doesn't have `Capture`.

Comment (marinafrydenb) [reply _includes/settlement-reconciliation.md]: fixed!

**Pull request merged**